USBHelperLauncher/Net/ApplicationEndpoint.cs

```diff
@@ -29,26 +30,50 @@ public void GetZipProxy(Session oS)
             var data = GetRequestData(oS);
             byte[] bytes = Convert.FromBase64String(data.Get("url"));
             string url = Encoding.UTF8.GetString(bytes);
-            string content;
-            using (WebClient client = new WebClient())
+
+            try
```

Comment:
Since this try catch only concerns `client.DownloadData(url);` it can be shrinked to contain that single line.